.github/workflows/cd-preview.yml

```diff
@@ -43,7 +40,13 @@ jobs:
           VERSION_NR=$(cat packages/browser/package.json | npx json version)
           echo "::set-output name=version-nr::$VERSION_NR"
       - run: npm run build
-      - run: npm run publish-preview -- --dist-tag "$TAG_SLUG" --yes
+      - run: |
+          # Unfortunately GitHub Actions does not currently let us do something like
+          #     if: secrets.NPM_TOKEN != ''
+          # so simply skip the command if the env var is not set:
+          if [ -n $NPM_TOKEN ]; then
+          npm run publish-preview -- --dist-tag "$TAG_SLUG" --yes
```

Comment:
Hmm, one problem I foresee here is that the `verify-imports-*` tasks that follow this will fail, because they're trying to install a package that wasn't published. We might need to fall back to just skipping the entire workflow if on a Dependabot branch and it wasn't triggered manually, if that's possible? Or even just if the trigger is Dependabot - it [looks like](https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#github-context) you can do something like `if: github.actor != 'dependabot'`?

Reply:
Hmm yeah you're right, that would be an issue. But wouldn't it be possible to use `if: github.actor != 'dependabot` everywhere, instead of testing for the presence of the secrets ? That would remove the non-portability of bash as well.

Reply:
Yes, that sounds like a good way forward to me.

Reply:
Oh, and with the ability to trigger the tests manually still :)